packages/application-config/src/validations.ts

```diff
@@ -42,11 +51,22 @@ const printErrors = (errors?: ErrorObject[] | null) => {
 };
 
 export const validateConfig = (
+  configType: ConfigType,
   config: JSONSchemaForCustomApplicationConfigurationFiles
 ): void => {
-  const valid = validate(config);
-  if (!valid) {
-    throw new Error(printErrors(validate.errors));
+  let validation;
+
+  if (configType === ConfigType.CUSTOM_APPLICATION) {
+    validation = validateCustomApplicationConfig;
+  } else if (configType === ConfigType.CUSTOM_VIEW) {
+    validation = validateCustomViewConfig;
```

Comment:
We need to validate differently the config file depending if it's for a Custom Application or a Custom View.